src/dlgTriggerEditor.cpp

```diff
@@ -8133,6 +8224,11 @@ bool dlgTriggerEditor::event(QEvent* event)
     return QMainWindow::event(event);
 }
 
+void dlgTriggerEditor::resizeEvent(QResizeEvent* event){
```

Comment:
Line-feed before the `{` please...

Reply:
Ooh, thought I had caught all the new areas and Beautified. Missed this one!